html/inc/boinc_db.inc

```diff
@@ -204,6 +204,11 @@ class BoincUser {
         $db = BoincDb::get();
         return $db->update($this, 'user', $clause);
     }
+    /**
+     * @param string|null $where_clause
+     * @param string|null $order_clause
+     * @return list<BoincUser>
+     */
```

Comment:
I added the typing. If we target recent php versions I can just add it directly in the function signature (except for the generic return).